pkg/chunk/chunk_test.go

```diff
@@ -35,22 +35,25 @@ func dummyChunk(now model.Time) Chunk {
 
 func dummyChunkForEncoding(now model.Time, metric labels.Labels, enc encoding.Encoding, samples int) Chunk {
 	c, _ := encoding.NewForEncoding(enc)
+	chunkStart := now.Add(-time.Hour)
```

Comment:
We pick now in input and then we manipulate it here. It's a bit hidden this manipulation. I would suggest to:
1. Rename the input `now` in `chunkStart`
2. Remove this line `chunkStart := now.Add(-time.Hour)`
3. In the places where you need a different start time, you can just call `dummyChunkForEncoding()` with the correct start time (ie. `now.Add(-time.Hour)`)

This way it will be more explicit.

Reply:
You are right that the implementation is not clean. Changing this function would mean changing its usage in about 30 places where it is used. I wanted to avoid doing too many changes in delete series PR by avoiding doing irrelevant changes. But since this is smaller PR, I think I can change it.

Reply:
> Changing this function would mean changing its usage in about 30 places where it is used.

Why do you need to change it across all existing tests as well, instead of just your new test?

Reply:
_Sorry, now I got it. It's unfortunate we've to carry on the legacy behaviour but I do understand it._